common/components/WalletDecrypt/WalletDecrypt.tsx

```diff
@@ -74,6 +74,7 @@ interface State {
   selectedWalletKey: WalletName | null;
   isInsecureOverridden: boolean;
   value: UnlockParams | null;
+  walletTimeoutID: number | undefined;
```

Comment:
@Sonicrida 

This should be `number | null` as an instance variable, because it doesn't affect the UI output. Also, undefined is semantically better for values that you haven't defined -- defining an `undefined`.

If you need to denote that a variable may not be coming in, you can use `exampleVariable?: number`. The `?` lets it know it may be `undefined`.